.gitlab-ci.yml

```diff
@@ -27,15 +27,15 @@ stages:
   tags:
     - nobatch
 
-.BuildKokkos:
+BuildKokkos:
```

Comment:
Do we need that change in that PR?

Reply:
I need this to build `Kokkos` and I could revert it after it has run once. I would prefer not to have to push two commits every time we want to update the CI but just build Kokkos every time (like we do in the rest of the CI).